### PR TITLE
Feature/fe 23 profile settings skeletion

### PR DESCRIPTION
**Pull request by aaman123**
### What this PR does (required):
-Implements Profile setting skeleton

### Screenshots / Videos (required):
- 
Screenshot (1546)

### Any information needed to test this feature (required):
- Please make a small navigation link to the profile route in order to test, right now there is no navigation set up from the dashboard itself ...

### Any issues with the current functionality (optional):
- Nope

### CODE REVIEW

**Comment by aaman123** [comment]
I sort of merged my previous commits in this branch in order to get the desired functionality, the changes for this issue is in the commit with ID "529adeb"

**Review by rajivtitus — COMMENTED** [review]
Hey @aaman123, it looks like a lot of the code in this PR has been reviewed in previous PRs. I figure that you have branched off a previous branch to work on other features, which is completely fine. However, please adjust the base of this PR so that only new changes are visible. Thanks

Changing the base branch of a pull request
- In the "Pull Requests" list, click the pull request you'd like to modify.
- Next to the pull request's title, click Edit.
- In the base branch drop-down menu, select the base branch you'd like to compare changes against.

File: client/src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "Edit Profile",
+        "Profile photo",
+        "Availability",
+        "Payment",
+        "Security",
+        "Settings",
+    ]
+
+    return(
+        <Grid container className={classes.root} spacing={2}>
```
Comment (rajivtitus) [inline client/src/pages/Profile/Profile.tsx:17]: Hi @aaman123 , as discussed on our meeting today, please make a functional sidebar here, which can be used to route to different components under the profile settings page :)

Comment (aaman123) [reply client/src/pages/Profile/Profile.tsx:17]: sure rajiv !

**Pull request merged**